web/src/beta/features/PluginPlayground/Plugins/presets/layers/hideFlyToDeleteLayer.ts

```diff
@@ -0,0 +1,240 @@
+import { FileType, PluginType } from "../../constants";
+import { PRESET_PLUGIN_COMMON_STYLE } from "../common";
+
+const yamlFile: FileType = {
+  id: "ui-hide-flyto-delete-layer-reearth-yml",
+  title: "reearth.yml",
+  sourceCode: `id: hide-flyto-delete-layer-plugin
+name: Hide Flyto Delete Layer
+version: 1.0.0
+extensions:
+  - id: hide-flyto-delete-layer
+    type: widget
+    name: Hide Flyto Delete Layer Widget
+    description: Hide Flyto Delete Layer Widget
+    widgetLayout:
+      defaultLocation:
+        zone: outer
+        section: center
+        area: top
+  `,
+  disableEdit: true,
+  disableDelete: true
+};
+
+const widgetFile: FileType = {
+  id: "hide-flyto-delete-layer-widget",
+  title: "hide-flyto-delete-layer.js",
+  sourceCode: `
+  
+const layerGeojson = {
+  type: "simple",
+  title: "sample",
+  visible: true,
+  data: {
+    type: "geojson",
+    url: "https://reearth.github.io/visualizer-plugin-sample-data/public/geojson/sample_polygon_polyline_marker.geojson"
+  },
+  polygon: {
+    fillColor: "red"
+  },
+  polyline: {
+    strokeColor: "red"
+  },
+  marker: {
+  }
+};
+
+const layer3dTiles = {
+  type: "simple", // Required
+  title: "3dtiles",
+  visible: true,
+  data: {
+    type: "3dtiles",
+    url: "https://assets.cms.plateau.reearth.io/assets/8b/cce097-2d4a-46eb-a98b-a78e7178dc30/13103_minato-ku_pref_2023_citygml_1_op_bldg_3dtiles_13103_minato-ku_lod2_no_texture/tileset.json" // URL of 3D Tiles
+  },
+  "3dtiles": {
+    // Settings for the 3D Tiles style.
+    pbr: false, 
+    selectedFeatureColor: "red" // If you select a feature, it will change color
+  }
+};
+
+reearth.layers.add(layerGeojson);
+reearth.layers.add(layer3dTiles);
```

Comment:
```suggestion
const pluginGeojsonLayerId = reearth.layers.add(layerGeojson);
const plugin3dTilesLayerId = reearth.layers.add(layer3dTiles);
```
We should be able to get the plugin layer ids in this way, and when do filter we don't need to hardcode the `1` `2` for preset layer ids.